bitrise.yml

```diff
@@ -0,0 +1,100 @@
+format_version: 4
```

Comment:
pls use the latest format version ([10](https://github.com/bitrise-io/bitrise/blob/master/models/models.go#L23)) to be able to benefit the new version lock (later)